ursa/graph/graph.py

```diff
@@ -36,46 +37,47 @@ def insert(self, key, oid, local_edges, foreign_edges, transaction_id):
 
         if key not in self.vertices:
             self.vertices[key] = \
-                [_Vertex(oid, local_edges, foreign_edges, transaction_id)]
+                [_Vertex(vertex_data, local_edges, foreign_edges,
+                         transaction_id)]
         else:
             temp_vertex = self.vertices[key][-1] \
-                .copy(oid=oid, transaction_id=transaction_id)
+                .copy(vertex_data=vertex_data, transaction_id=transaction_id)
             temp_vertex = temp_vertex.add_local_edges(
                 transaction_id, local_edges)
             temp_vertex = temp_vertex.add_foreign_edges(
                 transaction_id, foreign_edges)
             self.vertices[key].append(temp_vertex)
 
     @ray.method(num_return_vals=0)
-    def update(self, key, node, local_edges, foreign_edges, transaction_id):
-        """Updates the data for a node in the graph.
+    def update(self, key, vertex_data, local_edges, foreign_edges,
```

Comment:
I don't see an @param  declaration for local_edges.

Reply:
I accidentally deleted it.